packages/neon/neon/lib/src/utils/push_utils.dart

```diff
@@ -1,5 +1,7 @@
 part of '../../neon.dart';
 
+const largeIconSize = 256;
```

Comment:
Making it part of the theme doesn't make sense here as we don't have access to the context :woman_facepalming:

Reply:
Sure, but I wouldn't put it into the theme. In theory this size should depend on platform variables (See https://developer.android.com/reference/android/R.dimen#notification_large_icon_height). It will only be used by the push notifications.

Reply:
Ah no it makes no sense since you can't divide a size by another size. Using the Size class has no benefit.

Reply:
Somehow I remember using something like this once :thinking: 

Now it's in but we could have used:
```dart
const largeIconSize = 256;

final scale = largeIconSize / pictureInfo.size.longestSide;
final scaledSize = pictureInfo.size.longestSide * scale;
```

Reply:
Hm yeah that's something we could do, but this would depend on square icons, right? I'm not sure if every icon actually is square so we need to calculate the dimensions for each side separately. I might be wrong but I'd rather not make assumptions about the icons

Reply:
no we would again just pass the `scaledSize.width` and `scaledSize.height` along. they don't need to be the same.
internally the multiplication operator is just Size(width*scale, height *scale) so it wouldn't sqare up anything.

It's basically the same code just achieved in a different way and we are getting philosophical here.

Reply:
But scaledSize should be a double, no? I was assuming that longestSide returns the length of the longest side and not a new size where all sides have the length of longestSide?

Reply:
last line should be:

```dart
final scaledSize = pictureInfo.size * scale;
```

Yeah markdown isn't a real IDE sorry for the confusion.

Reply:
Ah I see. Could you open a PR with this small fix?